onnxruntime/python/tools/transformers/models/whisper/whisper_encoder_decoder_init.py

```diff
@@ -15,12 +15,12 @@
 import onnx
 import torch
 from transformers import WhisperConfig
-from whisper_decoder import WhisperDecoderInit
-from whisper_encoder import WhisperEncoder, WhisperEncoderInputs
-
 from onnxruntime import InferenceSession
 
+sys.path.append(os.path.join(os.path.dirname(__file__)))
```

Comment:
Maybe we can move them to `__init__.py` so that no need to update multiple files.